### PR TITLE
PMM-11961_node_related_agents

### PR DESCRIPTION
**Pull request by rasika-chivate**
https://jira.percona.com/browse/PMM-11961

As a PMM User I need to be able to see the list of agents which relate to a particular Node only (filtered by NodeID).

As a PMM User I want to be able to expand the row that relates to a Node and be able to see a more detailed information about the Node that may not be visible on the table view.

### CODE REVIEW

File: docs/details/dashboards/dashboard-inventory.md [hunk 1]
```diff
@@ -26,6 +28,10 @@ Each binary (exporter, agent) running on a client will get an `agent_type` value
 - `node_exporter` is assigned to an agent that extracts the node metrics
 - `mysqld_exporter` and `qan-mysql-perfschema-agent` are assigned to agents that extract metrics from mysql and its performance schema respectively.
 
+To view the agents running on a service and their health status, click **OK** under the **Monitoring** column. Furthermore, you can also check the properties of a particular agent by clicking the <image src="../../_images/dots-three-vertical.ico" width="15px" aria-label="triple dots" /> icon under the **Options** column.
```
Comment (filipmikes1) [inline docs/details/dashboards/dashboard-inventory.md:31]: Text in Monitoring column can be "OK" or "Failed" not only OK.

File: docs/details/dashboards/dashboard-inventory.md [hunk 2]
```diff
@@ -34,6 +40,18 @@ Each `node_id` is associated with a `machine_id` (from `/etc/machine-id`). Nodes
 
 By expanding the entry from the options column, you can check the node labels and attributes.
 
+Starting with PMM 2.38.0, you can see the number of agents running on any particular node. When you click on any node, the UI navigates to the view of agents, which is filtered to display only agents related to that specific node. 
+
+To see the details of the agents running, do the following:
+
+1. Select the node and click OK. A page that provides the user with crucial information regarding the total number of agents deployed is displayed.
```
Comment (filipmikes1) [inline docs/details/dashboards/dashboard-inventory.md:47]: Also not only OK but Failed value can be clickable as well.

File: docs/details/dashboards/dashboard-inventory.md [hunk 3]
```diff
@@ -34,6 +40,18 @@ Each `node_id` is associated with a `machine_id` (from `/etc/machine-id`). Nodes
...
+        ![!image](../../_images/PMM_Inventory_Node_Selection.png)
+
+2. Click on the down arrow under the Options column for further details about a specific agent.
+3. Select the node and click the down key to check details about an agent and their current health status.
```
Comment (filipmikes1) [inline docs/details/dashboards/dashboard-inventory.md:51]: What do you mean by clicking down key to check details?

Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-inventory.md]: @filipmikes1 updated it. Please check.

File: docs/details/dashboards/dashboard-inventory.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-inventory.md]: Updated it.

File: docs/details/dashboards/dashboard-inventory.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-inventory.md]: Updated it.

File: docs/details/dashboards/dashboard-inventory.md [hunk 4]
```diff
@@ -34,6 +40,18 @@ Each `node_id` is associated with a `machine_id` (from `/etc/machine-id`). Nodes
 
 By expanding the entry from the options column, you can check the node labels and attributes.
 
+Starting with PMM 2.38.0, you can see the number of agents running on any particular node. When you click on any node, the UI navigates to the view of agents, which is filtered to display only agents related to that specific node. 
+
+To see the details of the agents running, do the following:
+
+1.On the **Nodes** tab,  under the **Monitoring** column, click **OK** or **Failed** depending on the status of the node that you have selected. A page that provides the user with crucial information regarding the total number of agents deployed on that node is displayed.. A page that provides the user with crucial information regarding the total number of agents deployed on that node is displayed.
```
Comment (filipmikes1) [inline docs/details/dashboards/dashboard-inventory.md:47]: Sentence is duplicated:
A page that provides the user with crucial information regarding the total number of agents deployed on that node is displayed.

File: docs/details/dashboards/dashboard-inventory.md [hunk 5]
```diff
@@ -34,6 +40,18 @@ Each `node_id` is associated with a `machine_id` (from `/etc/machine-id`). Nodes
...
+        ![!image](../../_images/PMM_Inventory_Node_Selection.png)
+
+2. Click on the <image src="../../_images/dots-three-vertical.ico" width="15px" aria-label="triple dots" /> icon under the **Options** column to view the properties of a specific agent.
+3.  On the **Nodes** tab, under the **Options** column, click on the <image src="../../_images/dots-three-vertical.ico" width="15px" aria-label="triple dots" /> icon for the selected node to check the properties and the current health status of an agent.
```
Comment (filipmikes1) [inline docs/details/dashboards/dashboard-inventory.md:51]: Wrong image, should be arrow icon.

**Review by atymchuk — APPROVED** [review]

**Review by filipmikes1 — APPROVED** [review]

**Pull request merged**